### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by KevinDaGame**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,13 +7,15 @@ I'm thinking of renaming the project to prevent confusion with the already exist
 
 Minecraft Version
 ------------------
-VoxelSniper was built against the Bukkit 1.15.2 API.
+VoxelSniper was built against the Bukkit 1.18.1 API.
```
Comment (Lennart99) [inline README.md:10]: Why not 1.18.2, now that I think of it?

File: README.md [hunk 2]
```diff
@@ -7,13 +7,15 @@ I'm thinking of renaming the project to prevent confusion with the already exist
...
 
- - Minecraft 1.15.2 [Tested on SpigotMC, release 6.0.0]
+***We will not support anything below 1.16.5***
+
+ - Minecraft 1.18
 
 Compilation
 -----------
-Pre-compiled JARs are available in releases. Latest release is `6.0.0`.
+Pre-compiled JARs are available in releases..
```
Comment (Lennart99) [inline README.md:18]:
```suggestion
Pre-compiled JARs will be available in releases..
```
Perhaps change this, until there actually are releases available?

**Review by Lennart99 — APPROVED** [review]
I am really critical now, 1 more general question, and 1 small point.

File: README.md (hunk 1 above)
Comment (Lennart99) [inline README.md:10]:
```suggestion
VoxelSniper is being built against the Bukkit 1.18.1 API.
```

Comment (KevinDaGame) [reply README.md:10]: Because I lifted it before 1.18.2 was out. No point in updating if there is no need. I think it improves backwards compatibility

Comment (Lennart99) [reply README.md]: Should we make a draft PR to change this back when there is a release?

**Pull request merged**